app/src/pages/authoring/index.tsx

```diff
@@ -65,6 +97,23 @@ export default function AuthoringPage() {
           <Button w={240} loading={draftMutation.isLoading} onClick={createResource}>
             {`Create New Draft ${resourceType}`}
           </Button>
+          <Title order={3}>{`Start From an Existing ${resourceType}:`}</Title>
+          {artifacts ? (
+            <Select
+              label={`Select an existing ${resourceType} to clone`}
+              data={artifacts}
+              value={selectedArtifact}
+              onChange={setSelectedArtifact}
+            />
+          ) : (
+            <Text>No artifacts</Text>
+          )}
+          <Button
+            w={240}
+            loading={draftCloneMutation.isLoading}
+            onClick={cloneResource}
+            disabled={!selectedArtifact}
+          >{`Clone Draft ${resourceType}`}</Button>
```

Comment:
No need to use string interpolation here:

```suggestion
          >Clone Draft {resourceType}</Button>
```

I see there are some other buttons from other code that do this too, wanna just update those as well?


But also see above comment about use of the word "clone"